sum-of-multiples/example.js

```diff
@@ -0,0 +1,18 @@
+const Sum = (factors = [3, 5]) => {
+  let self = this instanceof Sum ? this : Object.getPrototypeOf(Sum);
```

Comment:
I think that this line is to use `Sum` as a constructor function (`new Sum()`) or just a function (`Sum()`).

I would write it as a class or just assuming it will be used as a constructor function:

```
class Sum {
  constructor(factors) {}
  to(limit) {}
}
```

or:

```
function Sum(factors) {
  this.factors = factors;
}
Sum.prototype.to = limit => {}
```